<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection_creator</class>
 <widget class="QDialog" name="Connection_creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection_creator</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="connection_types">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="IP">
     <attribute name="title">
      <string>IP</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1001</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="COM">
     <attribute name="title">
      <string>COM</string>
     </attribute>
     <widget class="QLabel" name="Com_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>COM</string>
      </property>
     </widget>
     <widget class="QComboBox" name="BRate_choose">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>72</width>
        <height>24</height>
       </rect>
      </property>
      <property name="currentText">
       <string>57600</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="BaudRateLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BaudRate</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PortsAvailableComboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>72</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Error_label_COM_tab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create connection</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
